website/src/pages/auth/signin.tsx

```diff
@@ -176,32 +189,40 @@ const EmailSignInForm = ({
   };
   return (
     <form onSubmit={handleSubmit(signinWithEmail)}>
-      <Stack>
-        <Input
-          type="email"
-          data-cy="email-address"
-          variant="outline"
-          size="lg"
-          placeholder="Email Address"
-          {...register("email")}
-        />
-        {enableEmailSigninCaptcha && (
-          <CloudFlareCaptcha
-            siteKey={cloudflareCaptchaSiteKey}
-            options={{ size: "invisible" }}
-            ref={captcha}
-            onSuccess={() => setCaptchaSuccess(true)}
-          ></CloudFlareCaptcha>
-        )}
-        <SigninButton
-          data-cy="signin-email-button"
-          leftIcon={<Mail />}
-          mt="4"
-          disabled={!captchaSuccess && enableEmailSigninCaptcha}
-        >
-          Continue with Email
-        </SigninButton>
-      </Stack>
+      <FormControl isInvalid={errors.email ? true : false}>
```

Comment:
Can you move this to wrap the Input and FormErrorMessage component only?

Reply:
Got it, I'm also adding FormControl and FormErrorMessage to the `pages/account/edit.tsx` file, if that's okay.

Reply:
> Got it, I'm also adding FormControl and FormErrorMessage to the `pages/account/edit.tsx` file, if that's okay.

Yeah, please add to edit file too